### ISSUE TITLE
出现问题 Setting `pad_token_id` to `eos_token_id`:106068 for open-end generation.

### ISSUE THREAD

**Issue by lin-cop** [OpenLMLab/MOSS#191]
========实例配置========
核数：16
内存：127 GB
磁盘：49% 9.7G/20G
显卡：NVIDIA A100-SXM4-80GB, 1
输出内容：
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:106068 for open-end generation.

**Comment by lin-cop** [comment]
![95d6abbb9ebe244a0ac50efab4e42fe](https://user-images.githubusercontent.com/131671223/234766512-9e93617f-f1fc-4724-8f1e-cdf0f4c6d330.png)

**Comment by lin-cop** [comment]
现在的教程太少，参考这个教程解决
[https://blog.csdn.net/genghaojie123/article/details/130357804](https://blog.csdn.net/genghaojie123/article/details/130357804)

**Comment by llx-08** [comment]
我是用多卡推理时也出现了这个问题
